cli/src/action/database/sqlite.rs

```diff
@@ -15,7 +15,8 @@
 //! Provides sqlite migration support to the database action
 
 use std::fs::{self, OpenOptions};
-use std::path::PathBuf;
+use std::os::unix::prelude::OpenOptionsExt;
```

Comment:
After further discussion, this will require some work to correctly set the permissions for other target families (e.g. Windows), so it may be best to leave this as a future compile error.